rules/common/src/main/kotlin/com/twitter/compose/rules/ComposeModifierComposable.kt

```diff
@@ -5,13 +5,13 @@ package com.twitter.compose.rules
 import com.twitter.rules.core.ComposeKtVisitor
 import com.twitter.rules.core.Emitter
 import com.twitter.rules.core.report
+import com.twitter.rules.core.util.isModifierReceiver
 import org.jetbrains.kotlin.psi.KtFunction
 
 class ComposeModifierComposable : ComposeKtVisitor {
 
     override fun visitComposable(function: KtFunction, autoCorrect: Boolean, emitter: Emitter) {
-        val receiverTypeReference = function.receiverTypeReference
-        if (receiverTypeReference == null || receiverTypeReference.text != "Modifier") return
+        if (!function.isModifierReceiver) return
```

Comment:
why this change?

Reply:
ah nvm didn't realize extension function

Reply:
It's all included in the new extension function: the null check and the Modifier check.